pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -396,6 +396,9 @@ func (c *InstallerController) manageInstallationPods(ctx context.Context, operat
 			if err != nil {
 				return true, 0, err
 			}
+			if installerPodFailed {
+				klog.Infof("Will retry %q for revision %d for the %s time because %s", currNodeState.NodeName, currNodeState.TargetRevision, nthTimeOr1st(newCurrNodeState.LastFailedCount), reason)
```

Comment:
except when we fail to update the spec a few lines down :)

Reply:
Well, the will is still correct.